packages/react-cards/src/components/Card/examples/Card.Compact.Example.tsx

```diff
@@ -2,59 +2,79 @@
 import * as React from 'react';
 import { Text } from '@uifabric/experiments';
```

Comment:
Isn't Text in OUFR now?

Reply:
Yes, let me update that as well. These examples were first created before that promotion. Thanks for catching this! 😄